finPath/nuxt.config.ts

```diff
@@ -1,6 +1,34 @@
+import vuetify, { transformAssetUrls } from "vite-plugin-vuetify";
 export default defineNuxtConfig({
-  ssr: false,
+  //...
```

Comment:
put ssr: false, as it will be client side application, ssr is not needed as of now.